classes/reportbase.php

```diff
@@ -324,4 +329,33 @@ public function get_middle_sql($actualrelation=true) {
         $whereparams = array_merge($whereparams, $eparams);
         return [$sql, $whereparams];
     }
+
+    /**
+     * set_additional_params.
+     *
+     * Sets the parameters to be supplied to the url to call the specific report page
+     */
+    public static function set_additional_params() {
+        $this->additional_params = ['optional' => [], 'required' => []];
```

Comment:
Uhm... `static` methods using `$this` is forbidden since long, long ago.

Reply:
You are right. This is because, to save time, I run pulls hurriedly before starting other tasks so that I can find the executed GHAs when I return to surveypro.